code/__DEFINES/dcs/signals.dm

```diff
@@ -290,38 +290,38 @@
 #define COMSIG_MOB_CLICK_RIGHT "mob_click_right"				//from base of mob/RightClickOn(): (atom/A)
 #define COMSIG_MOB_CLICK_SHIFT_RIGHT "mob_click_shift_right"	//from base of mob/ShiftRightClick(): (atom/A)
 #define COMSIG_MOB_CLICK_ALT_RIGHT "mob_click_alt_right"		//from base of mob/AltRightClick(): (atom/A)
-	#define COMSIG_MOB_CLICK_CANCELED (1<<0)
-	#define COMSIG_MOB_CLICK_HANDLED (1<<1)
+#define COMSIG_MOB_CLICK_CANCELED (1<<0)
+#define COMSIG_MOB_CLICK_HANDLED (1<<1)
 #define COMSIG_MOB_ATTACK_RANGED "mob_attack_ranged"			//from base of mob/RangedAttack(): (atom/A, params)
 #define COMSIG_MOB_THROW "mob_throw"							//from base of /mob/throw_item(): (atom/target)
 #define COMSIG_MOB_UPDATE_SIGHT "mob_update_sight"				//from base of /mob/update_sight(): ()
 #define COMSIG_MOB_HUD_CREATED "mob_hud_created"				//from base of mob/create_mob_hud(): ()
 
 #define COMSIG_MOB_ITEM_ATTACK "mob_item_attack"				//from base of /obj/item/attack(): (mob/target, /obj/item/attacking_item)
 #define COMSIG_MOB_ITEM_ATTACK_ALTERNATE "mob_item_attack_alt"	//from base of /obj/item/attack_alternate(): (mob/target, /obj/item/attacking_item)
-	#define COMPONENT_ITEM_NO_ATTACK (1<<0)						//return this if you dont want attacka and altattacks to run
+#define COMPONENT_ITEM_NO_ATTACK (1<<0)						//return this if you dont want attacka and altattacks to run
 
 #define COMSIG_MOB_ITEM_AFTERATTACK "mob_item_afterattack"		//from base of obj/item/afterattack(): (atom/target, mob/user, has_proximity, click_parameters)
 #define COMSIG_MOB_ITEM_AFTERATTACK_ALTERNATE "mob_item_afterattack_alternate" //from base of obj/item/afterattack_alternate(): (atom/target, mob/user, has_proximity, click_parameters)
 #define COMSIG_MOB_SAY "mob_say" 								// from /mob/living/say(): (proc args list)
 #define COMSIG_MOB_DEADSAY "mob_deadsay" 							// from /mob/living/say_dead(): (proc args list)
-	#define MOB_DEADSAY_SIGNAL_INTERCEPT (1<<0)
+#define MOB_DEADSAY_SIGNAL_INTERCEPT (1<<0)
 #define COMSIG_MOB_LOGIN "mob_login"							//from /mob/Login(): ()
```

Comment:
These changes to the return bitflag defines of signals should be undone
They are purposely indented once to show how they are return values

Reply:
Unadressed comment

Reply:
If you have a return bitflag, generally it would be better to have something similar to: `#define RETURN_MOB_DEADSAY_SIGNAL_INTERCEPT (1<<0)`
I can go through and rename the defines if you would prefer

Reply:
I meant the indentation as it creates a tree-like format thats easy to follow when looking at that the defines

-signal
   return_value_one
   return_value_two
  return_value_three
-signal
    return_value_one
   return_value_two
  return_value_three

Reply:
Yea sure, I'll revert the indentation removal for this

Reply:
I believe I reverted them all; lmk if I missed any